cypress/e2e/v1.0/accommodation/questions/accommodationStatus.ts

```diff
@@ -0,0 +1,223 @@
+
+      cy.saveAndContinue()
+
+      cy.assertStepUrlIs(stepUrl)
+      cy.getQuestion(question)
+        .getRadio(options.temporary)
+        .getConditionalQuestion()
+        .hasValidationError('Select the type of temporary accommodation')
+    })
+
+    typesOfTemporaryAccommodation.forEach(typeOfTemporaryAccommodation => {
+      it(`summary page displays "${options.temporary} - ${typeOfTemporaryAccommodation}"`, () => {
+        cy.visitStep(stepUrl)
+        cy.getQuestion(question).getRadio(options.temporary).clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .clickLabel()
+
+        cy.saveAndContinue()
+
+        cy.visitStep(summaryPage)
+        cy.getSummary(question).getAnswer(options.temporary).hasSecondaryAnswer(typeOfTemporaryAccommodation)
+      })
+    })
+
+    typesOfTemporaryAccommodation.forEach(typeOfTemporaryAccommodation => {
+      it(`validates the date "${options.temporary} - ${typeOfTemporaryAccommodation} - Expected end date"`, () => {
+        cy.visitStep(stepUrl)
+        cy.getQuestion(question).getRadio(options.temporary).clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .getConditionalQuestion()
+          .enterDate('99-99-9999')
+
+        cy.saveAndContinue()
+
+        cy.assertStepUrlIs(stepUrl)
+
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .getConditionalQuestion()
+          .hasValidationError('Enter a future date')
+      })
+    })
+
+    typesOfTemporaryAccommodation.forEach(typeOfTemporaryAccommodation => {
+      it(`summary page displays "${options.temporary} - ${typeOfTemporaryAccommodation} - Expected end date"`, () => {
+        cy.visitStep(stepUrl)
+        cy.getQuestion(question).getRadio(options.temporary).clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .clickLabel()
+        cy.getQuestion(question)
+          .getRadio(options.temporary)
+          .getConditionalQuestion()
+          .getRadio(typeOfTemporaryAccommodation)
+          .getConditionalQuestion()
+          .enterDate('01-01-2050')
+
+        cy.saveAndContinue()
+
+        cy.visitStep(summaryPage)
+        cy.getSummary(question).getAnswer(options.temporary).hasSecondaryAnswer('Expected end date:')
+        cy.getSummary(question).getAnswer(options.temporary).hasSecondaryAnswer('01 January 2050')
```

Comment:
should be able to chain these, but also we need a way of checking that these are the only secondary answers 🤔 might have to implement something similar to hasRadios() where hasSecondaryAnswers() takes an array

```suggestion
        cy.getSummary(question).getAnswer(options.temporary).hasSecondaryAnswer('Expected end date:').hasSecondaryAnswer('01 January 2050')
```